src/bot.ts

```diff
@@ -4,60 +4,232 @@ import { Bot, Context, InputFile, InlineKeyboard } from "grammy";
 const { BOT_TOKEN: token = "" } = process.env;
 export const bot = new Bot(token);
 
-//сховище для стоврення записки
+let isGetFile = false;
+let isDeleteNote = false;
+let isEditNote = false;
+
+//сховище для стоврення та зміни записки
 const memoData = {};
 
+// id для зміни файлу
+let editFileId = 0;
+
+const instance = axios.create({
+    baseURL: "https://sr-kpi-api-development.up.railway.app",
+});
```

Comment:
instance could be reused in toher parts of project. Move it to separate file

Reply:
Fixed